docs/resources/ecosystem/officials.md

```diff
@@ -8,7 +8,7 @@ These libraries are hosted in the [ZIO organization](https://github.com/zio/) on
 - [ZIO Actors](https://github.com/zio/zio-actors) — A high-performance, purely-functional library for building, composing, and supervising typed actors based on ZIO
 - [ZIO Akka Cluster](https://github.com/zio/zio-akka-cluster) — A ZIO wrapper for Akka Cluster
 - [ZIO Config](https://github.com/zio/zio-config) — A ZIO based configuration parsing library
-- [ZIO Web](https://github.com/zio/zio-web) — A ZIO-powered HTTP server and client
+- [ZIO Microservice](https://github.com/zio/zio-microservice) — A ZIO-powered microservices via HTTP and other protocols
```

Comment:
```suggestion
- [ZIO Microservice](https://github.com/zio/zio-microservice) — ZIO-powered microservices via HTTP and other protocols
```

Reply:
Can you also alphabetize?

Reply:
@adamgfraser I kept the `A` for consistency with the other entries in the list. They all start with `A`. So I would suggest all are dropped or we keep this one as well.

Reply:
Updated the order

Reply:
@mkows As currently written it is grammatically incorrect because "microservices" is being used as an adjective and there is no noun.

Reply:
Right. Good point.